build/signedbuild.yml

```diff
@@ -113,13 +113,13 @@ jobs:
     inputs:
       InputType: Basic
       # Double-quote syntax with trailing backslash on each line concatenates lines without spaces
-      AnalyzeTarget: "src\\AccessibilityInsights.Actions\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Automation\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Core\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Desktop\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Rules\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.RuleSelection\\bin\\Debug\\*.dll;\
-                      src\\AccessibilityInsights.Win32\\bin\\Debug\\*.dll;\
+      AnalyzeTarget: "src\\Axe.Windows.Actions\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Automation\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Core\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Desktop\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Rules\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.RuleSelection\\bin\\Debug\\*.dll;\
+                      src\\Axe.Windows.Win32\\bin\\Debug\\*.dll;\
                       src\\Axe.Windows.Telemetry\\bin\\Debug\\*.dll;"
```

Comment:
These are intended to be alphabetically sorted, so Telemetry should come before Win32 (be sure to update the trailing backslashes)